addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4300,6 +4300,16 @@ registry.ImageTools = SnippetOptionWidget.extend({
         $(document).on('mousedown', mousedown);
         $(document).on('mouseup', mouseup);
     },
+    async resetCrop() {
+        const cropper = new weWidgets.ImageCropWidget(this, this.$target[0]);
+        await cropper.appendTo(this.options.wysiwyg.$editable);
+        await cropper.reset();
+    },
+    async resetTransform() {
+        this.$target
+        .attr('style', (this.$target.attr('style') || '')
+        .replace(/[^;]*transform[\w:]*;?/g, ''));
+    },
```

Comment:
Missing comments and parameters (you can also fix the editor's team job by adding the ones they did not add)